go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/prometheus/client_golang v1.12.2
 	github.com/prometheus/client_model v0.2.0
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
+	github.com/redis/go-redis/v9 v9.4.0
```

Comment:
If we consolidate `redis-sentinel.go` and `redis.go`, we shouldn't need to import both `v8.11.5` and `v9.4.0` versions of effectively the same tool. Can we please test both redis cluster and redis sentinel with `v9.4.0` and keep only the updated version?